### DISCUSSION TITLE
Dracula's accent color and GTK themes

### DISCUSSION THREAD

**Discussion by RaphGL** [dracula/dracula-theme#408] [General] [answered]
### The Accent Color
I've been a huge fan of the Dracula theme, in fact I use it in almost everything I can and sometimes make it for those that don't have an official theme, but a problem that I often have with some themes is that some people choose pink or green as the accent color, which in my opinion have a big contrast with the background, which defeats the purpose of dark themes for me. I'd rather have the comment color(#6272a4) as the accent color. Not only does this make themes become inconsistent, since everybody is choosing different accent colors(when one is required) but it sometimes ruins some very well made themes for me. Here's some examples of this:
![image](https://user-images.githubusercontent.com/28673457/82219580-a1b2f400-9904-11ea-8e65-e6394b59f98f.png)
![image](https://user-images.githubusercontent.com/28673457/82219668-bee7c280-9904-11ea-86a7-480569d5896a.png)

In contrast with mine(biased opinion here):
![image](https://user-images.githubusercontent.com/28673457/82219735-d58e1980-9904-11ea-8a5a-ca0a54956165.png)
![image](https://user-images.githubusercontent.com/28673457/82219771-e2127200-9904-11ea-98a4-b022b7dce9f7.png)

So having a proper accent color and maybe specifying colors for other common UI elements so that every theme is more consistent instead of people having to make a lot of decisions based on the colors provided(that seem to assume that the theme will be used in syntax highlighting only) and personal choices.
### The GTK theme
It seems that Zeno Rocha has suggested to the Ant Dracula developer to move the theme here, but it's been a month since then and still nothing and there's also an issue opened here from 2017 about that but so far there hasn't been any moves towards making that happen. It would be great if we could get together and make one or fork/change the Ant Dracula theme to something that fits more with the Dracula theme and has a better accent color(sorry pink is a terrible accent color).

**Comment by benknoble** [comment]
I think maybe one topic per discussion would be better (yeah, ok, so I'm used to StackExchange).

Re: accent colors, there *is* a spec, though I believe it was intended more for syntax highlighting and editors (https://spec.draculatheme.com).

**Reply by RaphGL** [reply]
Yes, it's assuming that Dracula will be used only with syntax highlighting, having a specification for UI would be great too and maybe divide the [color palette section](https://draculatheme.com/contribute#color-palette) into Syntax and User Interface categories. If you look for the UI themes there's a lot of inconsistency and everybody is picking different accent colors.

**Reply by milouse** [reply]
Wow, I just discovered that there was a spec :O Thank you for pointing this out, it will help me a lot!

**Comment by zenorocha** [comment, marked as answer]
We officially have a GTK theme now 🎉  🎉  🎉 

https://draculatheme.com/gtk

Thanks @EliverLara for doing such a great job on this!

**Reply by RaphGL** [reply]
The theme is great, sadly it's using pink as the accent color, it sticks out too much, purple would be better for the eyes. For now I won't use it. Maybe the solution would be having different accent colors so people have a choice. Anyways great work everyone!

**Reply by zenorocha** [reply]
I agree purple would be a better choice. Can we make that happen @EliverLara?

**Reply by EliverLara** [reply]
@zenorocha that makes sense to me, purple looks great as accent color, I'll change it.
